package-lock.json

```diff
@@ -1837,9 +1838,9 @@
       }
     },
     "node_modules/@babel/runtime": {
-      "version": "7.16.7",
-      "resolved": "https://registry.npmjs.org/@babel/runtime/-/runtime-7.16.7.tgz",
-      "integrity": "sha512-9E9FJowqAsytyOY6LG+1KuueckRL+aQW+mKvXRXnuFGyRAyepJPmEo9vgMfXUA6O9u3IeEdv9MAkppFcaQwogQ==",
+      "version": "7.17.2",
```

Comment:
Not sure if version changes will cause issues -- @paulcretu what is our SOP on this?

Reply:
Yes I would try to avoid changing dependency versions in a PR unless it's necessary and related to the scope of the PR. A few reasons for this but the best reason is that if the upgrade breaks something, it's easier to revert a dedicated PR